metaflow/plugins/aip/tests/flows/flow_triggering_flow.py

```diff
@@ -104,22 +102,20 @@ def end(self):
         """Trigger downstream pipeline and test triggering behaviors"""
         if self.trigger_enabled:
             logger.info("\nTesting run_kubeflow_pipeline")
-            run_id, run_uid = run_argo_workflow(
-                KUBERNETES_NAMESPACE,
-                self.template_name,
+            argo_helper = ArgoHelper(KUBERNETES_NAMESPACE)
+            run_id, run_uid = argo_helper.template_submit(
```

Comment:
Now that we diverge from v1 `run_kubeflow_pipeline` and v2 `run_arg_workflow`, for consistency with `python flow.py aip trigger`, wouldn't this be trigger?

Reply:
We need to decide which one to align to I guess. The metaflow concept is `trigger` while the argo cli uses submit. I'll align with the metaflow command name as it's in metaflow code.